3_javascript/javascript1/week3/script3.js

```diff
@@ -0,0 +1,135 @@
+// Remove the item in names that is equal to nameToRemove
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+let remainingNames = names.filter(name => name !==nameToRemove);  
+  console.log(remainingNames); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+//   When will we be there??
+// Write a function where you speficy your speed in km/h and how far you have to go in km. 
+// The function has to return the time it will take to arrive at your destination. 
+// The time should be formatted like this: 
+// 3 hours and 34 minutes.
+
+function totalTravelTime (travelInformation) {
```

Comment:
Consider using a name like `calculateTravelDuration` which is a bit more explicit.